src/utils/queue.rs

```diff
@@ -4,6 +4,7 @@ use postgres::Connection;
 use error::Result;
 
 pub fn add_crate_to_queue(conn: &Connection, name: &str, version: &str, priority: i32) -> Result<()> {
+    ::web::metrics::TOTAL_BUILDS.inc();
```

Comment:
I'd move this incr to `queue`, after the package has been built/failed 5 times, otherwise if we have a long queue our metrics are going to be messed up.

Reply:
In other words, the counter would become "packages we've built", rather than "packages we've queued"? That seems fine to me; i could also see it diverging over time if we manually queued builds or reset their attempts counter, so this works for me.